### PR TITLE
I18n support for dashboard

### PR DESCRIPTION
**Pull request by h8570rg**
## Summary

I18n support for dashboard.

en
image

ja
image

## Checklist
- [x] [changeset file(s)](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) included

### CODE REVIEW

File: packages/app/src/pages/dashboard/endpoints/_/body/item/qrcode/index.tsx [hunk 1]
```diff
@@ -51,26 +53,21 @@ const QRCode: React.FC<Props> = ({ endpoint }) => {
           title={
             <div className="flex items-center gap-2">
               <QrcodeIcon className="w-em" />
-              <div>QR Code</div>
+              <div>{t('endpointQRCodeShare.title')}</div>
             </div>
           }
-          description="Share an endpoint with the QR code."
```
Comment (h8570rg) [inline packages/app/src/pages/dashboard/endpoints/_/body/item/qrcode/index.tsx:57]: deleted in new design

File: packages/app/src/locales/ja/common.json [hunk 2]
```diff
@@ -11,5 +11,47 @@
...
+    "groups": {
+      "title": "ダッシュボード",
+      "description": "追加された$t(endpoint)を一覧表示します"
+    }
+  },
+  "addEndpointButtonLabel": "$t(endpoint)の追加",
+  "addGroupButtonLabel": "$t(group)の追加",
+  "exportEndpoints": "$t(endpoint)リストのエクスポート",
+  "importEndpoints": "$t(endpoint)リストのインポート",
+  "enterEndpoint": "$t(endpoint)",
```
Comment (ejithon) [inline packages/app/src/locales/ja/common.json:35]: Perhaps this can be replaced with other words such as 登録 or 入力, and so on.

Comment (h8570rg) [reply packages/app/src/locales/ja/common.json:35]: You mean "enterEndpoint" is not always same as "$t(endpoint)" ?

This 'enterEndpoint' is used for the link to endpoint page, and currently the text is '管理画面' (short for '管理画面へ行く') in ja.

Is `"enterEndpoint": "管理画面",` better  than` "enterEndpoint": "$t(endpoint)"` ?

Comment (h8570rg) [reply packages/app/src/locales/ja/common.json]: fixed! 8833af0b4d57013c30851025c1c0f81f7f9ce0e9

**Review by ejithon — APPROVED** [review]

**Pull request merged**